### DISCUSSION TITLE
How to update manifests with multiple different package urls?

### DISCUSSION THREAD

**Discussion by borque** [microsoft/winget-pkgs#88224] [Q&A]
Hello!
This might be a stupid question, but how to update manifests with many different architectures and languages included?

For example Mozillas Firefox and Thunderbird have at least threee architectures (x86, x64, ARM), different installer types and many languages in the manifests.
As the latest updates for the Thunderbird- and FirefoxESR-releases were missing I tried to do these myself using >wingetcreate update< where I provided all the necessary information and -of course- all the new URLs after the --urls argument. After a longer pause wingetcreate exited telling me it could not find the matching URLs in the original manifest.
So I ended up by updating the manifests manually only for the english and german version which is IMHO better than nothing but a loss of quality though.

Am I making or understanding something wrong? How can these large manigests be updated easily?

Thanks for help!

**Comment by Trenly** [comment]
Using wingetcreate you can use `-i` to use interactive mode to map all the URLs. If you use the YamlCreate script, there is a `Quick Update` mode which requests the URL for each entry

**Comment by Jaifroid** [comment]
There's an issue https://github.com/microsoft/winget-create/issues/291 regarding the failure to update manifests with multiple and valid URLs. But it looks like wingetcreate is not receiving much love from the winget team...
